frontend/apps/reader/modules/readergesture.lua

```diff
@@ -550,6 +553,15 @@ function ReaderGesture:gestureAction(action)
         G_reader_settings:flipNilOrFalse("input_ignore_gsensor")
         Device:toggleGSensor()
         self:onGSensorToggle()
+    elseif action == "toggle_reflow" then
+        if self.document.configurable.text_wrap == 1 then
+            self.document.configurable.text_wrap = 0
+        else
+            self.document.configurable.text_wrap = 1
+        end
```

Comment:
might need to check for not credocument?

Reply:
True, since otherwise you'll set (and subsequently toggle) a nonsense value and send some no-op events. It's fairly harmless otherwise. :-)